common/nextpnr.cc

```diff
@@ -202,7 +202,7 @@ Property Property::from_string(const std::string &s)
     size_t cursor = s.find_first_not_of("01xz");
     if (cursor == std::string::npos) {
         // if it fits into an int64_t make it a number
-        if (s.size() <= 64) {
+        if (s.size() <= 256) {
```

Comment:
This can't be correct, because intval is only 64 bits long
![image](https://github.com/openXC7/nextpnr-xilinx/assets/148607/59068425-6163-4748-8237-c7e7eef2715e)